assets/base/i18n/en.json

```diff
@@ -465,6 +465,12 @@
   "login.signIn": "Log In",
   "login.signingIn": "Logging In",
   "login.username": "Username",
+  "logout.fail.cancel": "Cancel",
+  "logout.fail.continue_anyway": "Continue Anyway",
+  "logout.fail.message": "You’re not fully logged out. Some data may continue to be accessible to this device once the device goes back online. What do you want to do?",
+  "logout.fail.message.forced": "We could not log you out of the server. Some data may continue to be accessible to this device once the device goes back online.",
+  "logout.fail.ok": "OK",
+  "logout.fail.title": "Logout not complete",
```

Comment:
@cwarnermm I updated the texts. Let me know if I missed anything (apart of the retry button)